index.html

```diff
@@ -0,0 +1,31 @@
+<html>
+  <head>
+    <link href="style.css" rel="stylesheet" type="text/css" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Inter&display=swap"
+    />
+    <link rel="stylesheet" href="https://unpkg.com/leaflet/dist/leaflet.css" />
+    <script src="https://unpkg.com/leaflet/dist/leaflet.js"></script>
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <header>
+      <div>
+        <h3>John Doe</h3>
+      </div>
+      <nav>
+        <a href="#projects">Projects</a>
+        <a href="#contacts">Contacts</a>
+      </nav>
+    </header>
+    <div id="map"></div>
```

Comment:
This map should be in the <main></main> section of the html. Let's refactor this when you create a new PR

Reply:
I fixed my comment above, the `<main></main>` wrapper